install.sh

```diff
@@ -7,9 +7,9 @@ nvm_has() {
   return $?
 }
 
-if [ -z "$NVM_DIR" ]; then
-  NVM_DIR="$HOME/.nvm"
-fi
+nvm_install_dir() {
+  echo ${NVM_DIR:-"$HOME/.nvm"}
```

Comment:
As far as my google- and SO-fu goes, looks good. Didn't find any authoritative resource though.